specifications/api/swagger.yaml

```
@@ -3674,6 +3685,15 @@ components:
              containing helpful information.
         delegations: *transactionDelegations
         validity_interval: *ApiValidityInterval
+        reference_policy_script_template:
+          <<: *ScriptTemplateValue
+          description: |
+            Optional policy script template that could be used as a script reference
+            in another transaction.
+            In the current transaction the script will enrich the first output. The script
+            is constructed by replacing cosigner with policy public key of a given wallet.
+            In the future transactions the reference script can be used by a given
+            wallet multiple times by referencing the current transaction id and index=0.
```

```suggestion
            Optional policy script template that could be used as a script reference
            in another transaction.
            In the current transaction, the script will be included as
            the first output (`index = 0`).
            The script is constructed by replacing the cosigner
            with the policy public key of the wallet.
            In future transactions, the reference script can be used
            by any wallet multiple times
            by referencing the current transaction `id` and `index = 0`.
```

Minor changes in wording.

---

done, thanks